DOCS.md

```diff
@@ -38,21 +58,30 @@ Use this plugin for deploying a docker container application to AWS EC2 Containe
 * `mount_points` - Mount points from host to container, format is `sourceVolume containerPath readOnly` where `sourceVolume`, `containerPath` are strings, `readOnly` is string [`true`, `false`]
 * `volumes` - Bind Mount Volumes, format is `name sourcePath` both values are strings. Note with FARGATE launch type, you only provide the name of the volume, not the `sourcePath`
 * `efs_volumes` - Define EFS volume, format: `name efs-id root-directory`. Current configuration doesn't support encryption in transit.
-* `placement_constraints` - Ecs task definition placement constraints. Specify an array of constraints as a single string. Note that "distinctInstance" type can only be specified during run task or in service. Not inside a task definition. 
+* `placement_constraints` - Ecs task definition placement constraints. Specify an array of constraints as a single string. Note that "distinctInstance" type can only be specified during run task or in service. Not inside a task definition.
+
+New parameters:
+* `capacity_providers` - Defines capacity providers. Format: list of `base(int) weight(int) name`; base - designates how many tasks, at a minimum, to run on the specified provider; weight - designates how many tasks will be assigned to this provider from among of all tasks cap in comparition to other providers; name - capacity provider's name
+* `enable_execute_command` - Whether or not to enable the execute command functionality for the containers. Value is boolean [`true`, `false`]
+* `propagate_tags` - Specifies whether to propagate the tags from the task definition to the task. Value is boolean [`true`, `false`]
+* `platform_version` - The platform version the task should run. A platform version is only specified for tasks using the Fargate launch type. If one is not specified, the LATEST platform version is used by default
+* `dont_wait` - If set on `true` - this drone's step won't wait for all tasks to finish. Step ends execution when all tasks enter into `RUNNING` state. This also doesn't checks execution exit status. Default `false`
+* `ignore_execution_fail` - If set on `true` - drone's step won't fail on container's exit code !=0.
+* `task_timeout` - Timeout in seconds for task to successfully set all stages from `PROVISSIONING` to `STOPPED` or to `RUNNING` if `dont-wait` flag enabled. Default 300.
+* `task_kill_on_timeout` - When task reaches timeout - send kill signal to the containers. Deafult `true`
 
 
 ## Example
 
 ```yaml
 steps:
   - name: Deploy to ECS
-    image: pelotech/drone-ecs
+    image: ////

```

Comment:
I suspect that this will change to proper plugin image path

Reply:
Actually don't have idea for now what to put here. The plugin isn't supposed to be on dockerhub at least for now. And I don't want to put ECR url in public docs.